src/main/resources/templates/index.html

```diff
@@ -20,11 +20,19 @@ <h1>Lifehack team B</h1>
     <div class="projects">
         <h2>Projects to check out</h2>
         <div class="cards">
-                <div class="card">
-                    <img class="card-image" alt="timezones" src="../public/images/timezones/timezone_icon.jpg" th:src="@{/images/timezones/timezone_icon.jpg}"/>
-                    <div class="card-copy">Check the clock, everywhere</div>
-                    <a class="card-link" href="timezones/index.html" th:href="@{/timezones/}"/>Timezones</a>
-                </div>
+            <div class="card">
+                <img class="card-image" alt="timezones" src="../public/images/timezones/timezone_icon.jpg"
+                     th:src="@{/images/timezones/timezone_icon.jpg}"/>
+                <div class="card-copy">Check the clock, everywhere</div>
+                <a class="card-link" href="timezones/index.html" th:href="@{/timezones/}"/>Timezones</a>
+            </div>
+
+            <!--NYT TILFØJET-->
+            <div>
+                <a href="restaurantFinder/startPage.html" th:href="@{/restaurantFinder}"/>Restaurant Finder</a>
```

Comment:
Please add a card-image as the timezones example including a logo.

Reply:
made logo on index.html.